test/unit/gateways/cyber_source_test.rb

```diff
@@ -263,6 +279,26 @@ def test_successful_credit_card_capture_request
     assert response_capture.test?
   end
 
+  def test_capture_includes_local_tax_amount
+    authorization = "1000;1842651133440156177166;AP4JY+Or4xRonEAOERAyMzQzOTEzMEM0MFZaNUZCBgDH3fgJ8AEGAMfd+AnwAwzRpAAA7RT/;authorize;100;USD;"
```

Comment:
Does it seem to you like one ought to be able to mock up an auth response and extract an authorization from it?? It did to me too, but I could not get it to work. Since we're specifically just checking that the correct fields are included in the request here, I didn't think it was worth any more time to keep banging on it.